examples/convert_ieeg_to_bids.py

```diff
@@ -43,92 +42,99 @@
 import shutil
-from pprint import pprint
 
-import numpy as np
+import matplotlib.pyplot as plt
 
 import mne
 from mne_bids import (write_raw_bids, BIDSPath,
                       read_raw_bids, print_dir_tree)
 
-
 # %%
 # Step 1: Download the data
 # -------------------------
 #
 # First, we need some data to work with. We will use the
 # data downloaded via MNE-Python's ``datasets`` API:
 # :func:`mne.datasets.misc.data_path`
-misc_path = mne.datasets.misc.data_path(force_update=True)
+misc_path = mne.datasets.misc.data_path()
 
 # The electrode coords data are in the tsv file format
 # which is easily read in using numpy
-fname = misc_path + '/ecog/sample_ecog_electrodes.tsv'
-data = np.loadtxt(fname, dtype=str, delimiter='\t',
-                  comments=None, encoding='utf-8')
-column_names = data[0, :]
-info = data[1:, :]
-electrode_tsv = dict()
-for i, name in enumerate(column_names):
-    electrode_tsv[name] = info[:, i].tolist()
-
-# load in channel names
-ch_names = electrode_tsv['name']
-
-# load in the xyz coordinates as a float
-elec = np.empty(shape=(len(ch_names), 3))
-for ind, axis in enumerate(['x', 'y', 'z']):
-    elec[:, ind] = list(map(float, electrode_tsv[axis]))
+raw = mne.io.read_raw_fif(op.join(
+    misc_path, 'seeg', 'sample_seeg_ieeg.fif'))
+raw.info['line_freq'] = 60  # specify power line frequency as required by BIDS
+subjects_dir = op.join(misc_path, 'seeg')  # Freesurfer recon-all directory
 
 # %%
-# Now we make a montage stating that the iEEG contacts are in the MNI
-# coordinate system, which corresponds to the `fsaverage` subject in
-# FreeSurfer. For example, one can look at how MNE-Python deals with iEEG data
-# at `Working with SEEG
+# Now we make a montage in ACPC space. MNE stores channel locations
+# in the "head" coordinate frame, which has the origin at the center
+# between the left and right auricular points. BIDS requires that iEEG data
+# be in ACPC space so we need to convert from "head" to "mni_tal" which is
+# an `ACPC-aligned coordinate system
+# <https://surfer.nmr.mgh.harvard.edu/fswiki/CoordinateSystems>`_.
+# The MNI Talairach coordinate system is very useful for group analysis as
+# shown in `Working with SEEG
 # <https://mne.tools/stable/auto_tutorials/misc/plot_seeg.html>`_.
-montage = mne.channels.make_dig_montage(ch_pos=dict(zip(ch_names, elec)),
-                                        coord_frame='mni_tal')
-print(f'Created {len(ch_names)} channel positions')
-print(dict(zip(ch_names, elec)))
+# The ``fsaverage`` template brain is in MNI space so we can use the Talairach
+# transform to get to this space from "mri" or Freesurfer surface RAS space.
+# So, first we have to use the ``trans`` we found from assigning or estimating
+# fiducial points (see `Locating Intracranial Electrode Contacts
+# <https://mne.tools/dev/auto_tutorials/clinical/10_ieeg_localize.html>`_),
+# and then we can use the Freesurfer Talairach transform to get to MNI space.
+
+# estimate trans
```

Comment:
"trans" seems like mne-python jargon. Maybe just "transformation from head to mri"?